root.pro

```diff
@@ -2,7 +2,11 @@ TEMPLATE = subdirs
 
 SUBDIRS = src tests type-to-cxx
 
-prf.path =  $$[QT_INSTALL_DATA]/mkspecs/features
```

Comment:
No need for qt4 support. Generally it's been removed on this project already.

Reply:
8583b9fc81656eb9f07ae9ad0d79e387ea2a6277